template.tpl

```diff
@@ -1138,6 +1165,236 @@ ___TEMPLATE_PARAMETERS___
         "type": "EQUALS"
       }
     ]
+  },
+  {
+    "type": "RADIO",
+    "name": "theTradeDeskTrackerOrUPixel",
+    "displayName": "Use Event Tracker ID vs. Universal Pixel ID",
+    "radioItems": [
+      {
+        "value": "tracker_id",
+        "displayValue": "Event Tracker ID"
+      },
+      {
+        "value": "upixel_id",
```

Comment:
if we decide to support only tracker_id (and import of only the GTM TTD static tracking tag) in initial delivered scope, will update this to omit upixel_id from the radio selection